actionview/test/ujs/public/test/attach-bindings.js

```diff
@@ -0,0 +1,8 @@
+document.addEventListener('rails:attachBindings', function() {
+  window.Rails.linkClickSelector += ', a[data-custom-remote-link]';
```

Comment:
Is this also part of the test framework harness? Why does it matter? Add a comment to explain it?

Reply:
I have also moved this file out of the test folder as it contains some setup for before requiring the library and not actual tests.